CONTRIBUTING.md

```diff
@@ -9,6 +9,28 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 2. Wait for the Jenkins build to finish
 3. Create a PR in the [buildbot](https://github.com/netlify/buildbot) to bump [the version](https://github.com/netlify/buildbot/blob/ddbb47739f5b85c954aad9dc3823ab0676432957/Jenkinsfile#L35) of the `build-image`.
 
+## Development
+
+### Linting
+
+[ShellCheck](https://github.com/koalaman/shellcheck) usage is recommended, however it is not enforced.
```

Comment:
I took the liberty of adding this as I consider it a useful tool. Let me know if you have a different view on the subject @Benaiah @ehmicky 👍

Reply:
Sounds good!